account_invoice_pricelist_sale/models/account_invoice.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import models, api
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    @api.multi
+    def onchange_partner_id(
```

Comment:
This can be part of the original account_invoice_pricelist module, not here, as this is not functionality attached to sales.